deepinv/sampling/langevin.py

```diff
@@ -53,6 +52,8 @@ class ULA(BaseSampling):
     :param float crit_conv: Threshold for verifying the convergence of the mean and variance estimates.
     :param bool verbose: prints progress of the algorithm.
 
+
```

Comment:
Since merging #397 there are some more files with refs to be added in sampling e.g. see https://github.com/deepinv/deepinv/tree/main/deepinv/sampling/sampling_iterators